### DISCUSSION TITLE
project abandoned?

### DISCUSSION THREAD

**Discussion by JaminShanti** [jamulussoftware/jamulus#3096] [General]
Looking at the commit rate, project looks abandon?
https://github.com/jamulussoftware/jamulus/graphs/contributors?from=2023-01-01&to=2023-07-07&type=c

**Comment by JaminShanti** [comment]
should we do a retro?  What drove out the active developers?  Sometimes purity tests can cause this.

**Reply by chrisrimple** [reply]
Not sure what you're talking about.  While it's true that contributions have slowed in the last few months, that's primarily due to decreased usage of the application as the Covid pandemic has subsided and musicians have returned to in-person rehearsals and performances, with less need for Jamulus.  Even so, there is still plenty of activity going on, as evidenced by the recent Discussion threads here.

**Reply by dcorson-ticino-com** [reply]
As Chris said.
And also, we have a mature application that does what we need without fuss and never crashes.
The recent work has been more to do with keeping up with changing platforms than with adding failing functionality.
The one or two functions that might be nice to have, such as more channels per connection or separate controls for left and right channels, will need major code rewrites. 
You are welcome to contribute if something causes you so much pain that you find it unusable.
The rest of us are using and enjoying this mature product almost daily, which is a big contribution to our well-being in these unsure times.

**Reply by pcar75** [reply]
As above , plus I think most developers are in the northern hemisphere , so in summer and past COVID, people are out, enjoying a warm season (hopefully not torid or in flames) :-o

EDIT : Or that Github now belongs to Microsoft since 2022 ?

**Comment by LarryAndrew** [comment]
Does anybody know ..does the jamulus signal route through resident soundcard in a desktop. and can it contribute to noise

**Reply by dcorson-ticino-com** [reply]
Hi Larry,
Jamulus can use internal and external sound cards.
How this is done depends on the operating system you use and is documented on www.jamulus.io.
Of course, any particular sound card can add noise if it is poorly made.

**Comment by JaminShanti** [comment]
This was some of my attempts, initially though I could help with the build process or work in on the UI, but I found the QT configuration on windows blocking,  I though back then if this was easier to do I could proceed with my original directions.  
https://github.com/orgs/jamulussoftware/discussions/1998
https://github.com/orgs/jamulussoftware/discussions/2745

**Reply by ann0see** [reply]
Ok. I think this means that we should help you getting the windows build running? Probably the documentation is not good enough for windows?

**Reply by JaminShanti** [reply]
Well that's a good question. Am I the only one using Windows? If so, that's fine. Just notate it that it's only supported on Mac. Would have saved me a lot of time. I said I don't own a Mac.

**Reply by dcorson-ticino-com** [reply]
As far as I am aware the great majority of development is on Linux and Windows.
Download the QT Creator and the Windows sources.
You need to set a couple of directory pointers in the Creator but I don't remember any particular difficulties.
(It has been a while 'though)

**Reply by ann0see** [reply]
@JaminShanti I've just tried to get a mingw build running and it worked. See https://github.com/orgs/jamulussoftware/discussions/1998#discussioncomment-6408233

**Reply by ann0see** [reply]
I mostly develop on Linux. However, development should be possible on all desktop OS.

**Reply by mcfnord** [reply]
Has anyone built starting from vanilla Windows recently?

**Reply by ann0see** [reply]
If you consider not having mingw installed probably that's what I did yesterday?

**Reply by JaminShanti** [reply]
I will try soon, I think you're suggestion is reasonable. Also, the QT version has increased so maybe that will solve it too. I tried with qt 5 and 6. I found that software quite hard to install.

**Reply by ann0see** [reply]
Qt? You need an account and be aware not to download the enterprise but the open source version. 

They do hide the open source installer very well.

**Reply by pljones** [reply]
@mcfnord 
> Has anyone built starting from vanilla Windows recently?

You mean _not_ using QT Creator?  It's harder work and I can't be bothered.  Using QT Creator makes it easier.  If I want the build that Github is going to produce for Windows, I use Github to do the work.

**Reply by ann0see** [reply]
I also did a Visual Studio based build and everything seemed to work fine after I ran windeployqt. So that's not really an issue.

**Reply by henkdegroot** [reply]
I am always using Windows and not run into issues recently, but true I did not setup a vanilla windows system recently.
I rarely built using QT Creator, just use that for dev and work on GUI item, I use the windows powershell script to generate/built most of the time.

**Comment by LarryAndrew** [comment]
Thank you for yourresponse .. I,m using a audio interface . does that replace the internal sound card in the desktop/laptop ..  BYW  how bout u answer this question directly instead of refering I read the Unclear discription in Jamulus
________________________________

**Reply by ann0see** [reply]
@LarryAndrew  please open a separate discussion e.g. here: https://github.com/orgs/jamulussoftware/discussions/new?category=general such that this remains focused.
